includes/acl/core/memory_utils.h

```diff
@@ -167,19 +167,34 @@ namespace acl
 		template<typename Type>
 		struct safe_underlying_type<Type, false> { using type = Type; };
 
+		template<typename DstType, typename SrcType, bool isreal = false>
```

Comment:
rename `isreal` to `is_real` to match the coding standard or the surrounding code